src/main/java/com/paneedah/mwc/groovyscript/script/CraftingStation.java

```diff
@@ -91,12 +91,12 @@ public static class RecipeBuilder extends AbstractRecipeBuilder<GSCrafting> {
 
         @Override
         public String getErrorMsg() {
-            return "Error adding Modern Warefare Cubed Workbench Recipe: "+this.name;
+            return "Error adding Modern Warfare Cubed Workbench Recipe: " + this.name;
```

Comment:
Displaying the name of the mod should be done using the constant in `ProjectConstants`